src/Shared/USConverters/USAssetAllocationsConverter.ts

```diff
@@ -0,0 +1,99 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Kamil Musialowski
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import MorningstarConverter from '../MorningstarConverter';
+import type { XRayUSConverterOptions } from '../../XRayUS/XRayUSOptions';
+import type XRayUSJSON from '../../XRayUS/XRayUSJSON';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class USAssetAllocationConverter extends MorningstarConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: XRayUSConverterOptions
+    ) {
+        super(options);
+    }
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override parse (
+        options: XRayUSConverterOptions,
+        hasMultiple?: boolean
+    ): void {
+        const table = this.table,
+            columnSuffix = hasMultiple ? `_${options.json.PortfolioName}` : '',
```

Comment:
Unnecessary duplicate of `type` in multi-XRay: 

<img width="313" height="51" alt="Image" src="https://github.com/user-attachments/assets/2d8583f7-e91d-4a89-afb6-0da8fe84bebd" />